les/fetcher.go

```diff
@@ -46,7 +46,7 @@ type lightFetcher struct {
 	peers           map[*peer]*fetcherPeerInfo
 	lastUpdateStats *updateStatsEntry
 
-	lock       sync.Mutex // qwerqwerqwe
+	lock       sync.Mutex
```

Comment:
Couldn't you replace `qwerqwerqwe` with some documentation about specifically what asset this lock protects?

Reply:
done